test-blink.cpp

```diff
@@ -32,19 +35,49 @@ int main(void)
         return 1;
     }
 
-    wiringPiSetupGpio();
-    pinMode(gpio, OUTPUT);
+    // Setup GPIO with libgpiod
+    struct gpiod_chip *chip;
+
+    chip = gpiod_chip_open("/dev/gpiochip0");
+    if (!chip)
+    {
+        std::cerr << "Failed to open gpiochip: /dev/gpiochip0" << std::endl;
+        return 1;
+    }
+
+    line = gpiod_chip_get_line(chip, gpio);
+    if (!line)
+    {
+        std::cerr << "Failed to get line! Output code: " << envGPIO << std::endl;
+        return 1;
+    }
+
+    int ret = gpiod_line_request_output(line, CONSUMER, 0);
+    if (ret < 0)
+    {
+        std::cerr << "Request line as output failed! Output code: " << ret << std::endl;
+        return 1;
+    }
 
     for (;;)
     {
-        digitalWrite(gpio, HIGH);
+        setGpioLineValue(line, 1);
         std::cout << "On" << std::endl;
-        delay(500);
+        usleep(5e5);
 
-        digitalWrite(gpio, LOW);
+        setGpioLineValue(line, 0);
         std::cout << "Off" << std::endl;
-        delay(500);
+        usleep(5e5);
     }
 
     return 0;
 }
+
+void setGpioLineValue(struct gpiod_line *line, int value)
+{
+   int ret = gpiod_line_set_value(line, value);
+   if(ret < 0)
+   {
+        std::cerr << "Failed to set line to " << value << "! Output code: " << ret << std::endl;
```

Comment:
```suggestion
        std::cerr << "Failed to set gpio line to " << value << std::endl;
```
As previous comments.